runner/Makefile

```diff
@@ -44,7 +56,6 @@ docker-build-ubuntu:
 	${DOCKER} build \
 	  --build-arg TARGETPLATFORM=${TARGETPLATFORM} \
 	  --build-arg RUNNER_VERSION=${RUNNER_VERSION} \
-	  --build-arg RUNNER_CONTAINER_HOOKS_VERSION=${RUNNER_CONTAINER_HOOKS_VERSION} \
```

Comment:
@toast-gear I think we can safely remove this arg for the rootless dind runner as the runner container hooks are installed only on the non-dind runner image.